SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1447 @@
+		$ppmEnvironmentalDetails.Add($result)
+		
+		##Retrieve NSX-T Manager password policy details
+		$nsxtManagerDetails = Get-NsxtServerDetail -fqdn $ppmVariables.sddcManagerFqdn -username $ppmVariables.sddcManagerUser -password $ppmVariables.sddcManagerPass -domain $ppmVariables.sddcDomainName -listNodes
+		foreach ($nsxtManagerNode in $nsxtManagerDetails.nodes) {
+			$result = getNsxtManagerPasswordPolicy  -server $nsxtManagerNode.fqdn -user $nsxtManagerDetails.adminUser -pass $nsxtManagerDetails.AdminPass
+			$result.productAttributes | Add-Member -notepropertyname "driftAlarm" -notepropertyvalue ""
+			$result.productAttributes | Add-Member -notepropertyname "driftMessage" -notepropertyvalue ""
+			$ppmEnvironmentalDetails.Add($result)
+		}		
+		##Retrieve NSX-T Edge password policy details
+		$results = getNsxtEdgeNodePasswordPolicy -server $nsxtManagerDetails.fqdn -user $nsxtManagerDetails.adminUser -pass $nsxtManagerDetails.AdminPass
+		foreach ($result in $results) {
+			$result.productAttributes | Add-Member -notepropertyname "driftAlarm" -notepropertyvalue ""
+			$result.productAttributes | Add-Member -notepropertyname "driftMessage" -notepropertyvalue ""
+			$ppmEnvironmentalDetails.Add($result)
+		}
+		##Retrieve WSA Password policy details
+		if ($ppmVariables.wsaFqdn -and $ppmVariables.wsaAdminUser -and $ppmVariables.wsaAdminPass) {
+			Write-LogMessage -type INFO -Message "Retrieving Workspace ONE Access server password policy..."
+			$result = getWSAPasswordPolicyAll -server $ppmVariables.wsaFqdn -user $ppmVariables.wsaAdminUser -pass $ppmVariables.wsaAdminPass
+			$result.productAttributes | Add-Member -notepropertyname "driftAlarm" -notepropertyvalue ""
+			$result.productAttributes | Add-Member -notepropertyname "driftMessage" -notepropertyvalue ""
+			$ppmEnvironmentalDetails.Add($result)
+			Write-LogMessage -type INFO -Message "Retrieving Workspace ONE Access server password policy...complete"
+		} else {
+			Write-LogMessage -Type WARNING -Message "Workspace ONE Access Skipped.  Login details are not provided."  
+		}
+
+		$ppmVariables.isEnvDetailSet = $true
+		Write-LogMessage -type INFO -Message "Retrieving Environment Details...Completed" -Colour Green
+	}
+}
+
+################################################################################
+####   Print current environment password policies settings as html file	####
+################################################################################
+Function exportReportHTML {
+	Param (
+		[Parameter (Mandatory = $true)] [ValidateNotNullOrEmpty()] [psobject]$ppmVariables,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmStandardConfigValues,
+		[Parameter (Mandatory = $true)] [AllowEmptyCollection()][System.Collections.Generic.List[System.Object]]$ppmEnvironmentalDetails,
+		[Parameter (mandatory = $false)] [Switch]$doComparison = $false
```

Comment:
`compare` might have been better parameter switch term.

Reply:
addressed in bbfbaf6dac6bb885aa39e716c9d8971c0a5a58d7